public/components/marketing-pricings/2.html

```diff
@@ -0,0 +1,148 @@
+<div class="mx-auto max-w-4xl px-4 py-8 sm:px-6 sm:py-12 lg:px-8">
```

Comment:
It might need to be a little wider as there's 3, `max-w-5xl` could work but maybe larger.

Reply:
This isn't done.